packages/ember/index.js

```diff
@@ -311,9 +311,7 @@ Ember.Instrumentation = {
 
 // ****@ember/runloop****
 
-// Using _globalsRun here so that mutating the function (adding
-// `next`, `later`, etc to it) is only available in globals builds
-Ember.run = runloop._globalsRun;
+Ember.run = runloop.run;
```

Comment:
The comment being removed here seems still applicable. I don't want `runloop.run` to have `.next` and friends on it...

Reply:
So I'm not sure if this is a concern anymore. My thinking was this would have been if you were trying to monkeypatch Ember, because _internally_ Ember used `run.next` and such. We don't do that anymore, we use `next` directly, so it shouldn't matter if someone adds methods to the internal run, correct?

Reply:
@pzuraq - IMHO, it is definitely still a concern. You **absolutely should not** be able to this:

```js
import { run } from '@ember/runloop';

run.next(function() {
  console.log('trolol');
});
```

And I'm fairly sure you can (depending on if `window.Ember` has been touched) with the changes in this PR.